braindecode/datasets/base.py

```diff
@@ -142,6 +144,88 @@ def transform(self, value):
         self._transform = value
 
 
+class SequenceDataset(WindowsDataset):
+    """Dataset of sequences of consecutive windows.
+
+    Parameters
+    ----------
+    windows : mne.Epochs
+        Windows obtained through the application of a windower to a BaseDataset
+        (see `braindecode.datautil.windowers`).
+    description : dict | pandas.Series | None
+        Holds additional info about the windows.
+    transform : callable | None
+        On-the-fly transform applied to a window before it is returned.
+    seq_len : int
+        Number of consecutive windows in a sequence.
+    step_len : int
+        Number of windows between two consecutive sequences.
+    label_transform : callable | None
+        On-the-fly transform to apply to the sequence of labels. Useful to e.g.
+        select a single label for each sequence.
+    on_missing : 'raise' | 'warn' | 'ignore'
+        What to do if no sequences can be extracted given the number of epochs
+        and sequence parameters. Valid keys are 'raise' | 'warn' | 'ignore'.
+        Default is 'raise'. If on_missing is 'warn' it will proceed but warn,
+        if 'ignore' it will proceed silently.
+    """
+    def __init__(self, windows, description=None, transform=None, seq_len=21,
+                 step_len=1, label_transform=None, on_missing='raise'):
```

Comment:
how about `target_transform` instead of `label_transform`?

Reply:
are we consistent about this naming in braindecode?

Reply:
I think we mostly use "target", so `target_transform` makes more sense.

Reply:
It's also how they name it in torchvision ;)